### DISCUSSION TITLE
iCasa ICZB-RM11S

### DISCUSSION THREAD

**Discussion by timoline** [Koenkk/zigbee2mqtt#5448] [Devices]
Hi,
is this image correct?
https://www.zigbee2mqtt.io/devices/ICZB-RM11S.html

I think it should be...
https://www.partsnl.nl/iczb-rm11s-zigbee-afstandsbediening?gclid=Cj0KCQiAoab_BRCxARIsANMx4S47aAG8qsY18Uco7Y3VY8W5NPb1t3n0_j1eBUT5VX-b--uCPOLVRw4aAg_-EALw_wcB

**Comment by Koenkk** [comment]
Fixed

**Comment by timoline** [comment]
I have another question about this remote.
Binding etc works the 4 knobs are also working fine .(actually 4 on and 4 off buttons)
But i can't see these buttons in exposes?

![image](https://user-images.githubusercontent.com/680252/103455338-768ee080-4cec-11eb-96fc-1e8ccdb45057.png)

How can we get this?

When I press these 4 buttons, I don't see any debug messages

**Reply by Koenkk** [reply]
If you bind the remote to another device the coordinator won't receive messages anymore.

**Comment by timoline** [comment]
Ok, thanx , I think I understand, didn't expected though.
I was hoping for some kind of virtual remote ☺️
